packages/accounts-base/accounts_server.js

```diff
@@ -37,6 +37,14 @@ export class AccountsServer extends AccountsCommon {
       loggedInUser: ['profile', 'username', 'emails'],
       otherUsers: ['profile', 'username']
     };
+
+    // use object to keep the reference when used in functions
+    // where _defaultPublishFields is destructured into lexical scope
+    // for publish callbacks that need `this`
+    this._defaultPublishFields = {
+      loggedInUser: ['profile', 'username', 'emails'],
```

Comment:
I would prefer to use the MongoDB standard here, using a map with fieldName: 1 to select fields.